sqlglot/dialects/athena.py

```diff
@@ -132,6 +132,7 @@ class Generator(Trino.Generator):
         TRANSFORMS = {
             **Trino.Generator.TRANSFORMS,
             exp.FileFormatProperty: lambda self, e: f"format={self.sql(e, 'this')}",
+            exp.PartitionedByProperty: lambda self, e: f"partitioned_by={self.sql(e, 'this')}",
```

Comment:
Note that the actual reason for the uppercasing is because the `Properties.NAME_TO_PROPERTY` map hardcodes everything to uppercase.

Since we already had a transform to lowercase `exp.FileFormatProperty`, I added one for `exp.PartitionedByProperty` since most of the properties in `Properties.NAME_TO_PROPERTY` arent relevant for Athena